### PR TITLE
Change into cow theme

### PR DESCRIPTION
**Pull request by anxolin**
Separate our Gnosis Protocol fee from a new theme for the COW thingy:

image

Now we can freely adapt our styles for the app. 

Here I just changed 1 color:
image

### CODE REVIEW

**Review by biocom — APPROVED** [review]

File: src/custom/theme/index.tsx [hunk 1]
```diff
@@ -1,103 +1,2 @@
...
-export const ThemedGlobalStyle = createGlobalStyle`
-  // Uniswap default
-  ${ThemedGlobalStyleUniswap}
-
-  html {
-    background-image: ${({ theme }) => `linear-gradient(0deg, ${theme.bg1} 0%, ${theme.bg2} 100%)`};
-  }
-`
+export { default } from './cowSwapTheme'
+export * from './cowSwapTheme'
```
Comment (alfetopito) [inline src/custom/theme/index.tsx:2]: Doesn't `export *` already export `default`?

**Review by alfetopito — COMMENTED** [review]
Soft approval.
Rather not have this merged while the colors look like this, though.
It'll make develop unusable :eye: :fire:

File: src/custom/theme/index.tsx (hunk 1 above)
Comment (anxolin) [reply src/custom/theme/index.tsx:2]: apparently nup

**Comment by anxolin** [comment]
Good point, I'll leave the branch open. And the first PRs of the COW thing, we can do it against this branch.

Do you think is good @alfetopito @W3stside @biocom

**Pull request closed without merging**